build/cli.cjs

```diff
@@ -12978,6 +13191,39 @@ async function powersOfTauImport(params, options) {
     // TODO Verify
 }
 
+async function powersOfTauImportNoOrigin(params, options) {
```

Comment:
Please add doc.

Reply:
Doc added in [new commit](https://github.com/MystenLabs/snarkjs/pull/1/commits/b5f6cf3ba828f3fbb8cf4100e7a1c1ec6408d7d9#diff-180b3d2af89416b99563fff8c37eb8ca1ef7591fc1ddd736b5dacf740ccd3760R787-R799).